.github/workflows/ci.yml

```diff
@@ -26,6 +45,13 @@ jobs:
           npm install && npm run build
           git diff --quiet || (echo "::error file=grammar.js::Generated parser differs from the checked in version" && exit 1)
 
-      - name: Parser tests 
+      - name: Parser tests
         shell: bash
         run: npm install && npm run test
+
+      - name: Smoke test
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        shell: bash
+        run: |
+          export PATH="$PATH:node_modules/.bin/"
```

Comment:
I believe you can avoid this and use `npm exec`, i.e.

```bash
npm exec -c 'tree-sitter --help'
```

Reply:
Yea. Let me try that.